src/settings.py

```diff
@@ -311,7 +311,6 @@ def regen_ssowatconf(setting_name, old_value, new_value):
         app_ssowatconf()
 
 
-@post_change_hook("tls_passthrough_enabled")
```

Comment:
Maybe we should have a `post_change_hook` to auto-regen the firewall conf when `trusted_proxies_list` changes ?